tst/schedulers/multiobjective/test_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import numpy as np
+
+from syne_tune.optimizer.schedulers.multiobjective.utils import (
+    hypervolume,
+    hypervolume_cumulative,
+)
+
+
+def test_hypervolume():
+    ref_point = np.array([2, 2])
+    points = np.array([[1, 0], [0.5, 0.5], [0, 1], [1.5, 0.75]])
+    hv = hypervolume(results_array=points, reference_point=ref_point)
+    assert hv == 3.25
+
+
+def test_hypervolume_progress():
+    ref_point = np.array([2, 2])
+    points = np.array([[1, 0], [0.5, 0.5], [0, 1], [1.5, 0.75]])
+    hv = hypervolume_cumulative(results_array=points, reference_point=ref_point)
+    assert np.allclose(hv, np.array([0.0, 2.0, 2.75, 3.25]))
```

Comment:
Is the `allclose` assertion because it's floating-point and thus non-deterministic? In the `test_hypervolume` test, why is it then able to be a strict equals check `hv == 3.25`?

Reply:
This is for floating point precision. Updated the other test too, good catch.